xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/vue/LivedataPagination.vue

```diff
@@ -71,12 +72,13 @@
 
     <!--
       The actual pagination widget
-      It dislays the the available pages numbers, and change to them on click.
+      It displays the the available pages numbers, and change to them on click.
       Not all page numbers are show depending of the `pagination.maxShownPages`
       property in the Livedata meta config.
       Arrows can be shown to go to first, last, previous, next page.
     -->
-    <nav class="pagination-indexes">
+    <nav class="pagination-indexes"
+      :aria-label="$t('livedata.pagination.index.label')">
```

Comment:
We can nest nav elements, there's no real incorrect way to use navs, as long as they describe a section with links.
> The [nav](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-nav-element.html#the-nav-element) element [represents](http://www.w3.org/TR/2011/WD-html5-20110525/rendering.html#represents) a section of a page that links to other pages or to parts within the page: a section with navigation links.

Nesting nav elements: https://stackoverflow.com/questions/6986612/in-html5-can-you-nest-nav-elements
Nav element def: https://www.w3.org/TR/2011/WD-html5-author-20110809/the-nav-element.html

___
The usage here is a bit of a stretch since the parent nav does not contain any other links. I think this would be better for users  to remove this nav.
There is some info relevant to navigation in the parent, even if not links, and removing the nested nav would just force users to navigate in the nav-related infos before accessing the links.

___
_Note_: 
As stated in the nav [definition](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-nav-element.html
), 
> Not all groups of links on a page need to be in a nav element — the element is primarily intended for sections that consist of major navigation blocks. In particular, it is common for footers to have a short list of links to various pages of a site, such as the terms of service, the home page, and a copyright page. The footer element alone is sufficient for such cases; while a nav element can be used in such cases, it is usually unnecessary.

Our use of the nav landmark here is not mandatory, however I think it's still a good idea to keep it because often livedata tables are central to the content of the page and it's nice to access their nav quickly. 
This nav landmark could be removed once we use another landmark close enough (such as a `section` for the liveData object).

Reply:
I agree that having some nav element in the LiveData pagination is probably fine, but I think we should reduce it to one.

Reply:
Another problem here is the label in combination with multiple Live Data elements on a single page. For example, consider the case of opening the attachments tab on a document that already contains a Live Data (I think there are also documents with multiple Live Data elements in the content, I just don't have any example right now).